src/env.h

```diff
@@ -371,6 +371,7 @@ struct ContextInfo {
   bool is_default = false;
 };
 
+
```

Comment:
Can you undo the whitespace change here and on line 839?

Reply:
Sorry, missed them and will remove them now. Thanks